### PR TITLE
fix debug build

### PR DESCRIPTION
**Pull request by smartprogrammer93**

### CODE REVIEW

**Review by LukaszRozmej — APPROVED** [review]

**Review by MarekM25 — APPROVED** [review]

**Review by benaadams — APPROVED** [review]

**Comment by rubo** [comment]
Hold on with this. This should be done using matrix. Will do, once recover my access.

**Review by FalcoXYZ — APPROVED** [review]
LGTM

**Review by rubo — CHANGES_REQUESTED** [review]
@smartprogrammer93 The workflow you modified wasn't completed successfully. Moreover, it has constantly failed for 2 days. Looks like GitHub made some changes in its runners. I'd suggest reverting workflow changes in this PR to merge it quickly.

@LukaszRozmej The workflow fix will likely require changes in the PR required actions list.

**Comment by smartprogrammer93** [comment]
@rubo it is not working because it is running out of storage space. As suggested ill remove the workflow change to be able merge it

File: .github/workflows/build-solutions.yml [hunk 1]
```diff
@@ -41,3 +42,9 @@ jobs:
         run: dotnet build EthereumTests.sln -c ${{ env.BUILD_CONFIG }} --no-restore
       - name: Build Benchmarks.sln
         run: dotnet build Benchmarks.sln -c ${{ env.BUILD_CONFIG }} --no-restore
+      - name: Build Nethermind.sln
+        run: dotnet build Nethermind.sln -c ${{ env.SECOND_BUILD_CONFIG }} --no-restore
+      - name: Build EthereumTests.sln
+        run: dotnet build EthereumTests.sln -c ${{ env.SECOND_BUILD_CONFIG }} --no-restore
+      - name: Build Benchmarks.sln
+        run: dotnet build Benchmarks.sln -c ${{ env.SECOND_BUILD_CONFIG }} --no-restore
```
Comment (LukaszRozmej) [inline .github/workflows/build-solutions.yml:50]: Shall we put it as a separate job, so it will run in parallel?

Comment (rubo) [reply .github/workflows/build-solutions.yml:50]: > The workflow fix will likely require changes in the PR required actions list.

That's what I mean.

**Review by rubo — APPROVED** [review]

**Pull request merged**